terraform/modules/tf_stream_alert/variables.tf

```diff
@@ -23,44 +23,36 @@ variable "rule_processor_config" {
   default = {}
 }
 
-variable "rule_processor_lambda_config" {
-  type    = "map"
-  default = {}
-}
+variable "rule_processor_version" {}
 
-variable "rule_processor_versions" {
-  type    = "map"
-  default = {}
-}
+variable "rule_processor_memory" {}
+
+variable "rule_processor_timeout" {}
 
 variable "alert_processor_config" {
   type    = "map"
   default = {}
 }
 
-variable "alert_processor_lambda_config" {
-  type    = "map"
-  default = {}
-}
+variable "alert_processor_version" {}
 
-variable "alert_processor_versions" {
-  type    = "map"
-  default = {}
-}
+variable "alert_processor_memory" {}
+
+variable "alert_processor_timeout" {}
 
 variable "output_lambda_functions" {
-  type    = "map"
-  default = {}
+  type    = "list"
```

Comment:
It is not necessary to specify `type` here, based on [terraform document](https://www.terraform.io/docs/configuration/variables.html) `If this field is omitted, the variable type will be inferred based on the default. If no default is provided, the type is assumed to be string.`